lib/mix/lib/mix/tasks/compile.ex

```diff
@@ -11,7 +11,7 @@ defmodule Mix.Tasks.Compile do
   ## Configuration
 
     * `:compilers` - compilers to run, defaults to `Mix.compilers/0`,
-      which are `[:yecc, :leex, :erlang, :elixir, :xref, :app]`.
+      which are `[:yecc, :leex, :erlang, :elixir, :xref, :app, :protocols]`.
```

Comment:
Oh, sorry. I thought it is because of `Enabled compilers: yecc, leex, erlang, elixir, xref, app, protocols` from `mix compile --list` command.